config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/' => 'greetings#hello'
```

Comment:
Rails [has a handy helper](https://guides.rubyonrails.org/routing.html#using-root) to create the root route. This has the added benefit that you can then make use of the `root_url` and `root_path` helpers throughout your codebase.